docs/download.md

```diff
@@ -18,18 +18,18 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
 that uses the RAPIDS Accelerator For Apache Spark. See the [getting-started
 guide](https://nvidia.github.io/spark-rapids/Getting-Started/) for more details.
 
-## Release v23.06.0
+## Release v23.08.0
 Hardware Requirements:
 
 The plugin is tested on the following architectures:
 
-	GPU Models: NVIDIA P100, V100, T4 and A2/A10/A30/A100 GPUs
+	GPU Models: NVIDIA P100, V100, T4, A2/A10/A30/A100, L4 and H100 GPUs
 
 Software Requirements:
 
 	OS: Ubuntu 20.04, Ubuntu 22.04, CentOS 7, or Rocky Linux 8
 
-	CUDA & NVIDIA Drivers*: 11.x & v470+
+	NVIDIA Driver*: 470+
 
 	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.2.2, 3.2.3, 3.2.4, 3.3.0, 3.3.1, 3.3.2, 3.4.0 Databricks 10.4 ML LTS or 11.3 ML LTS Runtime and GCP Dataproc 2.0, Dataproc 2.1
```

Comment:
```suggestion
	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.2.2, 3.2.3, 3.2.4, 3.3.0, 3.3.1, 3.3.2, 3.4.0, 3.4.1, Databricks 10.4 ML LTS or 11.3 ML LTS Runtime and GCP Dataproc 2.0, Dataproc 2.1
```
This is so long, it probably would be better as a bulleted list where all the patch versions of a minor release are on a line, e.g.:
```
Supported Spark Versions:
- Apache Spark 3.1.1, 3.1.2, or 3.1.3
- Apache Spark 3.2.0, 3.2.1, 3.2.2, 3.2.3, or 3.2.4
- ...
```